functions/adoptopenjdk-java.bash

```diff
@@ -0,0 +1,22 @@
+adoptopenjdk_install_apt(){
+  local adoptKey="/tmp/adoptopenjdk.asc"
+  if ! dpkg -s 'adoptopenjdk-8-hotspot-jre' >/dev/null 2>&1; then # Check if already is installed
+    cond_redirect systemctl stop openhab2.service
+    echo -n "$(timestamp) [openHABian] Installing AdoptOpenJDK... "
+    cond_redirect update-alternatives --remove-all java
+    wget -qO "$adoptKey" https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public
+    if cond_redirect apt-key add "$adoptKey"; then echo "OK"; else echo "FAILED (keyserver)"; exit 1; fi
+    rm -f "$adoptKey"
+    echo "deb http://adoptopenjdk.jfrog.io/adoptopenjdk/deb buster main" > /etc/apt/sources.list.d/adoptopenjdk.list
+    cond_redirect apt-get update
+    if cond_redirect apt-get --yes install adoptopenjdk-8-hotspot-jre; then echo "OK"; else echo "FAILED"; exit 1; fi
+    cond_redirect systemctl start openhab2.service
+  fi
```

Comment:
do we need another update-alternatives --java or will Adopt install take care?

Reply:
The install takes care of it.